basic/model.py

```diff
@@ -220,17 +220,17 @@ def _build_forward(self):
             encoded_question = tf.concat(2, [fw_outputs, bw_outputs])
             if config.share_lstm_weights:
                 tf.get_variable_scope().reuse_variables()
-                # [batch_size, num_sentences, sentence_size, 2*hidden_size]
+                # [batch_size, num_sentences, sentence_size, 2*hidden_size] (verified)
```

Comment:
i manually checked the shapes of these as a sanity check, i should remove these comments...